redaxo/src/addons/cronjob/install.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+rex_sql::factory()->setQuery('
+    CREATE TABLE IF NOT EXISTS `'.rex::getTable('cronjob').'` (
```

Comment:
In R5.4 kann man es ja eigentlich so schreiben:

```php
rex_sql_table::get(rex::getTable('cronjob'))
    ->ensureColumn(new rex_sql_column('id', 'int(10) unsigned', false, null, 'auto_increment'))
    ->ensureColumn(new rex_sql_column('name', 'varchar(255)', true))
    ->ensureColumn(new rex_sql_column('description', 'varchar(255)', true))
    ->ensureColumn(new rex_sql_column('type', 'varchar(255)', true))
    ->ensureColumn(new rex_sql_column('parameters', 'text', true))
    ->ensureColumn(new rex_sql_column('interval', 'text'))
    ->ensureColumn(new rex_sql_column('nexttime', 'datetime', true))
    ->ensureColumn(new rex_sql_column('environment', 'varchar(255)'))
    ->ensureColumn(new rex_sql_column('execution_moment', 'tinyint(1)'))
    ->ensureColumn(new rex_sql_column('execution_start', 'datetime'))
    ->ensureColumn(new rex_sql_column('status', 'tinyint(1)'))
    ->ensureColumn(new rex_sql_column('createdate', 'datetime'))
    ->ensureColumn(new rex_sql_column('createuser', 'varchar(255)'))
    ->ensureColumn(new rex_sql_column('updatedate', 'datetime'))
    ->ensureColumn(new rex_sql_column('updateuser', 'varchar(255)'))
    ->setPrimaryKey('id')
    ->ensure();
```

(bzw. noch mit den neuen Shortcut-Methoden..)

Das funktioniert, egal, ob die Tabelle schon existiert etc. Es mangelt somit meiner Meinung nach nicht an neuen Methoden. Die würden uns im Fallback-Fall ja auch gar nichts nützen, da wir dort mit R5.3-Mitteln auskommen müssen. Und dort ist `rex_sql_table` nun mal so beschränkt:
- Tabellen konnten nicht angelegt werden (kein ensure, kein create, nur alter)
- `::get()` schlug schon fehl für noch nicht existente Tabellen
- kein `->setPrimaryKey`

Daher müssen wir im Fallback-Fall die Tabelle zumindest schon mal händisch anlegen, mit korrekter Engine etc, und auch korrektem Primary Key.